src/components/transactions/TransactionList.vue

```diff
@@ -34,13 +34,9 @@
         <thead>
           <tr>
             <th scope="col">Action</th>
-            <th scope="col" width="12%">From</th>
-            <th scope="col">Sender</th>
-            <th scope="col">Txn hash</th>
-            <th scope="col">Block number</th>
-            <th scope="col" width="13%">Amount</th>
-            <th scope="col" width="12%">To</th>
-            <th scope="col">Receiver</th>
+            <th v-for="column in transactionsColumns" :key="column.key" scope="col">
```

Comment:
I don't recomend creating columns dinamically, it tends to be more complicated to mantain, harder to find the code, and a real pain to get the width correctly